### PR TITLE
New ReStructuredText Cheat Sheet

### PR DESCRIPTION
**Pull request by nickschafran**
https://duck.co/ia/view/restructuredtext_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@nickschafran Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: restructuredtext.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
+{
```
Comment (moollaza) [inline restructuredtext.json]: @nickschafran thanks for making this!

It looks like this file has been placed in the root of the repo though, which is causing problems.

Please move it to `share/goodies/cheat_sheets/json/`

File: restructuredtext.json [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    "name": "ReStructuredText",
+    "description": "ReStructuredText syntax",
+
+    "metadata": {
+        "sourceName": "ReStructuredText Documentaton",
+        "sourceUrl" : "http://docutils.sourceforge.net/docs/user/rst/quickstart.html"
+    },
+
+    "aliases": [
+        "restructuredtext", "rst", ".rst"
```
Comment (moollaza) [inline restructuredtext.json]: `restructuredtext` doesn't need to be an alias -- it's already the default trigger based on the filename :+1:  -- please remove it

File: share/goodie/cheat_sheets/json/restructuredtext.json [hunk 3]
```diff
@@ -0,0 +1,145 @@
+{
+    "id": "restructuredtext_cheat_sheet",
+    "name": "ReStructuredText",
+    "description": "ReStructuredText syntax",
+
+    "metadata": {
+        "sourceName": "ReStructuredText Documentaton",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/restructuredtext.json]: `SourceForge`

**Comment by alohaas** [comment]
@nickschafran, thanks again for working on a new reStructuredText Cheat Sheet! You’ve been doing a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in reStructuredText.

When you're ready to move on, please: 

- Create a new Instant Answer for searches you do related to reStructuredText or programming syntax. 

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Comment by moollaza** [comment]
@nickschafran Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@nickschafran this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=reStructuredText Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/restructuredtext_cheat_sheet_mobile.png)

**Pull request merged**